.gitignore

```diff
@@ -1,5 +1,3 @@
-.DS_Store
```

Comment:
Was there a reason this line was removed ? We generally want to include this in the .gitignore, to make sure it's not accidentally committed by a MacOS user !

Reply:
If I'm not mistaken, the code is actually contributed by us on a branch of the package that is unreleased. I don't think that it is subject to constraints, then ?
The alternative is to recode the functions (there is room for improvement).